.eslintrc.json

```diff
@@ -0,0 +1,14 @@
+{
+    "env": {
+        "commonjs": true,
+        "node": true
+    },
+    "parserOptions": {
+        "ecmaVersion": 2015
+    },
+    "rules": {
+        "space-before-function-paren": ["error", {"anonymous": "always", "named": "never", "asyncArrow": "always"}],
+         "no-console": "off"
+    },
+    "extends": "eslint:recommended"
+}
```

Comment:
I think we should keep a final newline char here?

Reply:
done